src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -45,28 +47,36 @@ export enum Status {
 }
 
 interface Props {
-  item: FormattedAction;
+  item;
```

Comment:
I feel it should be typed, i know that there will be a lot of "optonal" fields. But now we can puts here any object, and easily break the app

Reply:
I know. 

I just couldn't make it work properly the first time around so at one point I just gave up and strait removed it.

You're right that it should typed so I guess I'll give it one more  try then.